lib/jxl/encode_internal.h

```diff
@@ -140,6 +140,95 @@ constexpr std::array<unsigned char, 32> kContainerHeader = {
 constexpr std::array<unsigned char, 8> kLevelBoxHeader = {0,   0,   0,   0x9,
                                                           'j', 'x', 'l', 'l'};
 
+class JxlEncoderChunkedFrameAdapter {
```

Comment:
A comment on what this is for wouldn't hurt :-)

Reply:
Done.